common/source/docs/common-omnibusnanov6.rst

```diff
@@ -0,0 +1,140 @@
+.. _common-omnibusf4pro:
+
+==================
+Omnibus F4 Nano V6 (must be V6)
+==================
```

Comment:
not enough ===

Reply:
I dont understand...looking at other board documents there does not seem to be any standard for the length of ===== or -----
in fact, most of the ones in this document were copied directly from its parent existing omnibusf4pro document and are not my creation...

Reply:
try to compile the wiki, you will see the syntax warning. I will a commit to correct to other. Those are annonying as they flood the promp and make us miss the real issues.
I would prefer that the correction is done in this pr. But if you cannot (don't know, don't have time etc.) do it, I will do it later

Reply:
not interested in setting up to enable compiling, but did run ReText and cleaned up as much as I could...also, changed == and -- to match header lengths (ReText does not complain about those)...about as much as I can do at this point